index.js

```diff
@@ -138,6 +139,7 @@ function bootstrap(options) {
 
   app.use(userMiddleware);
   app.use(hofMiddleware.cookies());
+  app.use(markdown());
```

Comment:
yes good point about the markdown middleware being loaded before, I'll be interested to know where you move it